_docs/_latest/configure/notifier/index.md

```diff
@@ -2,4 +2,140 @@
 title: Notifier
 order: 300
 ---
+
 # {{ page.title }}
+
+Forseti Notifier can send a variety of notifications to alert you
+of Forseti events. These notifications can be configured to be sent
+to different channels, and in different formats.
+
+### Notification Types
+
+  1. Inventory summary
+  1. Scanner summary
+  1. Violations
+  1. CSCC findings
+  
+### Notification Channels
+
+  1. Email
+  1. Slack
+  1. GCS
+
+### Notification Formats
+
+  1. csv (human-readable data)
+  1. json (structured data)
+
+## Configuring Notifier
+
+### Inventory Summary (TBD)
+
+### Scanner Summary (TBD)
+
+### Violation Notifications
+
+1. Open `forseti-security/configs/forseti_conf.yaml`.
+1. Navigate to the `notifier` > `resources` section.
+
+The following options are available, on a per resource basis. You can mix and
+match any combination of notifiers for each resource.
+
+This example shows adding a email, slack, and gcs notifer on cloudsql violations.
+
+```
+notifier:
+    resources:
+        - resource: cloudsql_acl_violations
+          should_notify: true
+          pipelines:
+             - name: gcs_violations_pipeline
+               configuration:
+                 gcs_path: gs://path_to_foo_bucket
+             - name: email_violations_pipeline
+               configuration:
+                 sendgrid_api_key: foobar_key
+                 sender: forseti-notify@forsetisecurity.org
```

Comment:
`sender:` should have a more appropriate placeholder

Reply:
Can you please give a suggestion?  Anything would be great!